static/templates/project/prototypetask.html

```diff
@@ -0,0 +1,61 @@
+<md-subheader>
```

Comment:
this will be used for any milestone so renaming doesnt make sense, let's leave these as they are